src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1324,13 +1324,19 @@ public async Task<(long, object)> ModifyPeriodicBackup(TransactionOperationConte
             string databaseName, BlittableJsonReaderObject etlConfiguration)
         {
             UpdateDatabaseCommand command;
+            var databaseRecord = LoadDatabaseRecord(databaseName, out var _);
+            var connectionStringName = etlConfiguration["ConnectionStringName"].ToString();
 
             switch (EtlConfiguration<ConnectionString>.GetEtlType(etlConfiguration))
             {
                 case EtlType.Raven:
+                    if (databaseRecord.RavenConnectionStrings.TryGetValue(connectionStringName, out var rvnConnection) == false)
+                        ThrowUnknownConnectionStringName(connectionStringName);
                     command = new AddRavenEtlCommand(JsonDeserializationCluster.RavenEtlConfiguration(etlConfiguration), databaseName);
```

Comment:
We are deserializing the configuration here (`JsonDeserializationCluster.RavenEtlConfiguration(etlConfiguration)`). Use this instance and:
1. Fire `.Validate()` method and throw if there are any errors
2. Add new method in `abstract EtlConfiguration` called `.ValidateConnectionString(DatabaseRecord)`